framework/test/unit/modules/nft/endpoint.spec.ts

```diff
@@ -419,21 +405,37 @@ describe('NFTEndpoint', () => {
 				},
 			});
 
-			const expectedSupportedCollection = {
-				collectionIDs: supportedCollections.map(collection =>
-					collection.collectionID.toString('hex'),
-				),
-			};
+			await expect(endpoint.getSupportedCollectionIDs(context)).resolves.toEqual({
+				supportedCollectionIDs: [`${chainID.toString('hex')}********`],
+			});
+		});
 
-			await expect(endpoint.getCollectionIDs(context)).resolves.toEqual(
-				expectedSupportedCollection,
-			);
+		it('should return an array with supported collection IDs', async () => {
```

Comment:
same comment as prev test and please expand the description to state when does this case happen

Reply:
Is there a test desc format that am missing. To me the current desc describes the test. Kindly update the test desc and I'll incorporate it. Thanks